react-frontend/src/css/nav.css

```diff
@@ -31,6 +32,7 @@ header {
     align-items: center;
     margin: 0 -10px 0 0;
     color: #fff;
+    height: 65px;
```

Comment:
setting heights can run you into trouble. Use padding or something

Reply:
After experimentation I found that trying to use padding cause the bottom border to disappear, perhaps we can revisit this in a future styling ticket

Reply:
i would like this to be demo'd so i can see what you are talking about

Reply:
@patricksimonian of course we defer to your advice here, but in @ryleyw's defense, the [Design System CSS for header](https://developer.gov.bc.ca/Design-System/Header-Basic#css) defines height, so I presume he was just following that.

Reply:
Totally thanks for pointing that out :)